### ISSUE TITLE
Accessibility update to viewer toolbar 

### ISSUE THREAD

**Issue by adamjarling** [samvera-labs/openseadragon-react-viewer#51]
### Description
An accessibility audit revealed that OpenSeadragon Viewer toolbar options have missing aria-roles. This needs to be fixed.

### Done looks like
- OSD toolbar component is aria-* compliant.

**Comment by adamjarling** [comment]
Closed via https://github.com/samvera-labs/openseadragon-react-viewer/pull/50
